tests/serverpod_test_server/test_e2e/email_authentication_provider_test.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/config.dart';
+import 'package:serverpod_test_server/test_util/test_key_manager.dart';
+import 'package:test/test.dart';
+
+void main() {
+  var client = Client(
+    serverUrl,
+    authenticationKeyManager: TestAuthKeyManager(),
+  );
```

Comment:
Suggestion: Add at least one failure case test too! Using the wrong password does not give you a valid auth key.